lib/v2/nber/utils.js

```diff
@@ -0,0 +1,28 @@
+const cheerio = require('cheerio');
+
+async function get_html(url) {
+    const browser = await require('@/utils/puppeteer')();
```

Comment:
Data are fed thru API
`https://www.nber.org/api/v1/working_page_listing/contentType/working_paper/_/_/search`
Please use that instead